### DISCUSSION TITLE
Bulk or CLI loading of MPO multiple termination cables?

### DISCUSSION THREAD

**Discussion by tomrockwell** [netbox-community/netbox#12387] [General]
Looking for method to load multiple termination cables to NB. Is there one? Can anyone provide an example?

I see there is an issue with this as feature request https://github.com/netbox-community/netbox/issues/10319

TIA

**Comment by candlerb** [comment]
Are you talking about an MPO breakout cable, e.g. which has an MPO connector at one end and 12 separate LC connectors at the other?

**Reply by tomrockwell** [reply]
Yes, I should have said MPO breakout --- my specific case of fan-out of QSFP28 100GE switch ports to 4x SFP28 25GE server ports.

I assume that I can do POST to the API endpoint directly with this data, but wondering if there is a YAML bulk upload method working.
